DemoApp/DemoAppConfiguration.swift

```diff
@@ -28,7 +28,7 @@ enum DemoAppConfiguration {
     // This function is called from `DemoAppCoordinator` before the Chat UI is created
     static func setInternalConfiguration() {
         StreamRuntimeCheck.assertionsEnabled = isStreamInternalConfiguration
-        StreamRuntimeCheck._isBackgroundMappingEnabled = false
+        StreamRuntimeCheck._isBackgroundMappingEnabled = isStreamInternalConfiguration
```

Comment:
I think we can put this to `true` always for the demo app. Let's get as much feedback as possible in the next few weeks.